cql3/selection/abstract_function_selector.hh

```diff
@@ -35,11 +35,11 @@ protected:
 public:
     static shared_ptr<factory> new_factory(shared_ptr<functions::function> fun, shared_ptr<selector_factories> factories);
 
-    abstract_function_selector(shared_ptr<functions::function> fun, std::vector<shared_ptr<selector>> arg_selectors)
+    abstract_function_selector(shared_ptr<functions::function> fun, std::vector<shared_ptr<selector>> arg_selectors, bool reserve_extra_arg = false)
```

Comment:
I couldn't figure out what this new "reserve_extra_arg" parameter is supposed to do.